op.c

```diff
@@ -6046,42 +6046,100 @@ Perl_newBINOP(pTHX_ I32 type, I32 flags, OP *first, OP *last)
     return fold_constants(op_integerize(op_std_init((OP *)binop)));
 }
 
+/* 4 bits per hex char, highest bit position = 0,1-3 => 1; 4-7 => 2; ... */
```

Comment:
Here, consider inserting a comment that shows the reader how to self-demonstrate what's happening, *e.g.*:
```
/* To make evident, configure with `-DDEBUGGING`, build, run `./perl -Ilib -Dy t/op/tr.t`. */
```

Reply:
done